src/java/htsjdk/samtools/SAMFileReader.java

```diff
@@ -694,25 +694,6 @@ private void init(final InputStream stream, File file, final File indexFile, fin
         }
     }
 
-    /**
```

Comment:
While we're killing this, could you also kill `isGzippedSAMFile` and delegate to the SamStreams variant?